esp/scm/ldapenvironment.ecm

```diff
@@ -44,7 +44,7 @@ ESPenum OUMode : int
 ESPrequest LDAPCreateEnvironmentRequest
 {
     bool                                              CreateLDAPEnvironment(false);
-    bool                                              CreateK8sSecret(false);
+    bool                                              CreateK8sSecrets(false);
```

Comment:
I'm not sure exactly how ECM files work, but I thought there was some versioning mechanism we used when changing interfaces. This feels like it's going to break anyone using the prior interface.

Reply:
That's true, although this service is still brand new and in the proof-of-concept stage... so not sure it's necessary.

But changing the name of an input and adding fields to the output would normally both be versioned.

Reply:
@richardkchapman As Tony said, this is brand new code that nobody has looked at yet.